messageBus/amazonSQS.js

```diff
@@ -41,12 +41,12 @@ module.exports.poll = (queue) => {
   });
 };
 
-module.exports.done = (messages) => {
+module.exports.done = (queue, messages) => {
```

Comment:
Just a point of clarification - is this so that you can use the function for different queues?

Reply:
Yes - this is so that I can use the same methods regardless of the queue.